packages/terser/terser_minified.bzl

```diff
@@ -98,8 +98,7 @@ so that it only affects the current build.
     ),
     "terser_bin": attr.label(
         doc = "An executable target that runs Terser",
-        # NB: will be substituted with "@npm//@bazel/terser/bin:terser" in the pkg_npm
```

Comment:
Is this still true?

Reply:
Looks like it is... keep/edit the existing comment? I guess other spots such as 20 lines up don't though 🤷

Reply:
Yeah all of `//packages/foo` get swapped to `//@bazel/foo` which seems obvious enough, and we'd have to comment that in a lot of places... but maybe worth it